libraries/apollo-normalized-cache-incubating/src/commonMain/kotlin/com/apollographql/apollo3/cache/normalized/ClientCacheExtensions.kt

```diff
@@ -323,6 +326,8 @@ fun <T> MutableExecutionOptions<T>.doNotStore(doNotStore: Boolean) = addExecutio
  *
  * Default: false
  */
+@Deprecated("Will be removed in v4, where this is the default behavior", ReplaceWith(""))
+@ApolloDeprecatedSince(v3_7_5)
 fun <T> MutableExecutionOptions<T>.emitCacheMisses(emitCacheMisses: Boolean) = addExecutionContext(
```

Comment:
@BoD sorry I missed this but this doesn't work. On v3 replacing with `""` is not a solution. We need to keep this and start deprecating with v4

Reply:
Oh yeah you're right deprecating it here is very helpful. No worries!